src/dataDisplay/tableCell.jsx

```diff
@@ -31,6 +84,7 @@ class TableCell extends Component {
         const cellPrefix = 'table-cell';
         const containerClasses = ClassNames(
             'table-cell',
+            'table--cell',
```

Comment:
NAB: I totally understand why you are adding this className, but it sure does stink. Not a blocker.

Reply:
It's part of your inital changes/commits, I'll change it :)

Reply:
I didn't change this because I alreay saw that is needed to apply the styles/sticky functionality from the `<Table />`.